.github/workflows/test-stories.yaml

```diff
@@ -116,8 +116,7 @@ jobs:
           externals: |
             [
               "**/!(*.module).scss",
-              "packages/button/src/Button/*.scss",
-              "packages/component-library/components/Spacing/Base.module.scss"
+              "packages/components/**/*.css",
```

Comment:
Wasn't sure if we wanted it to be a blanket one, so only applied it to KAIO. Can revise.